packages/discovery-react-components/src/components/SearchFacets/SearchFacets.tsx

```diff
@@ -25,6 +25,10 @@ import { FallbackComponent } from 'utils/FallbackComponent';
 import { withErrorBoundary } from 'react-error-boundary';
 
 interface SearchFacetsProps {
+  /**
+   * ID for the SearchInput
+   */
+  id?: string;
```

Comment:
the way this is being used, its more of an `idPrefix` or `idSuffix`. `id` is a bit generic for this property since its only getting applied to a sub-component rather than the root wrapper object.

i would either:
a) attach this id to the root level and have it default to something if not set
b) append `prefix` or `suffix` to the name of this attribute so that its usage matches the name

Reply:
I'm going to go with an implementation similar to the `SearchInput` component, where the id prop passed in is attached to the root `div` of the component.  If nothing is passed in, a default is given.